src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -94,6 +98,37 @@ private static IEnumerable<CredentialCacheCount> GetCredentialCacheCounts()
             yield return CreateHostPortCredentialCacheCount();
             yield return CreateUriAndHostPortCredentialCacheCount();
         }
+        
+        public static IEnumerable<object[]> StandardAuthTypeWithNetworkCredential =>
+            new[]
+            {
+                new object[] {authenticationTypeNTLM, credential1},
+                new object[] {authenticationTypeKerberos, credential2},
+                new object[] {authenticationTypeNegotiate, credential3},
+                new object[] {authenticationTypeBasic, credential4},
+                
+                new object[] {authenticationTypeNTLM, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeKerberos, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeNegotiate, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                // Default credentials cannot be supplied for the Basic authentication scheme.
```

Comment:
@davidsh @CIPop Sorry I misunderstood some concepts. I want to say that in .Net framework, when authType == Digest, if WDigestAvailable == true, it will pass the validation; if WDigestAvailable == false, will throw ArgumentException. https://github.com/Microsoft/referencesource/blob/4fe4349175f4c5091d972a7e56ea12012f1e7170/System/net/System/Net/CredentialCache.cs#L68

While in .Net Core, WDigestAvailable will always be false (we don't support it). Therefore, it will always throw ArgumentException. So this is a behavior difference, if put them in the same test case, my concern is it may fail in some .Net Framework conditions.